src/App/DeliveryOptions/Service/DeliveryOptionsService.php

```diff
@@ -46,6 +46,7 @@ class DeliveryOptionsService implements DeliveryOptionsServiceInterface
         'priceSameDayDelivery'         => CarrierSettings::PRICE_DELIVERY_TYPE_SAME_DAY,
         'priceSignature'               => CarrierSettings::PRICE_SIGNATURE,
         'priceStandardDelivery'        => CarrierSettings::PRICE_DELIVERY_TYPE_STANDARD,
+        'priceCollect'                 => CarrierSettings::PRICE_COLLECT,
```

Comment:
hier moet de `allowExpressDelivery` ook nog naar de CarrierSettings gemapped worden!

Reply:
Is nu mee genomen!